rules/jsll/tests/helpers/common.ts

```diff
@@ -38,13 +38,41 @@ export const perfectConfig = {
     useShortNameForContentBlob: true
 };
 
+const calculatedConfig = `
+var appId = 'app' + 'id';
+var environment = 'test';
+var isprod = 'True';
+if (isprod.toLowerCase() === 'true') {
```

Comment:
I'm missing something or this if is equal to `if (true)`?

Reply:
@sarvaje Yes 😄 ,  this is a proof-of-concept code for `calculated` config property values, and I used the code from one of the websites I tested.

Reply:
ok!